### PR TITLE
Fruits <feature

### PR DESCRIPTION
**Pull request by wouterz**
more like banana for now, might add more fruits later

### CODE REVIEW

**Comment by DaanVermunt** [comment]
Clean implementation <feature>

File: core/src/main/java/sem/group15/bubblebobble/core/objects/Player.java [hunk 1]
```diff
@@ -160,6 +160,13 @@ public void handleCollision(final GameObject other) {
                     logger.log("Player touched bubble");
                 }
             }
+            if (other instanceof Fruit) {
+                Fruit fruit = (Fruit) other;
+                if (fruit.getAliveTime() > 0.5) {
```
Comment (DaanVermunt) [inline core/src/main/java/sem/group15/bubblebobble/core/objects/Player.java]: why does it have to be alive for this amount of time?

Comment (wouterz) [inline core/src/main/java/sem/group15/bubblebobble/core/objects/Player.java]: The fruit spawns at the location of the enemy so without this the fruit gets picked up instantly after popping a bubble and you never see it or have to collect it.  <review

**Comment by DaanVermunt** [comment]
Please fix merge conflicts <layout>

**Pull request merged**